topostats/tracing/disordered_tracing.py

```diff
@@ -21,7 +21,8 @@
 
 LOGGER = logging.getLogger(LOGGER_NAME)
 
-# pylint: disable=too-many-positional-arguments
+# too-many-positional-arguments
+# pylint: disable=R0917
```

Comment:
Ah nice! I've been looking for a way to suppress this warning as the `# pylint: disable=too-many-positional-arguments` doesn't seem to work!

Reply:
Its quite annoying as its a valid name but doesn't seem to be recognised. I should raise an issue but still have to report to the Topoloy developers the issues with Python version releases and other problems uncovered the other day and don't have time for everything.